scripts/nesi_scripts/run_adv_im_obs_maui.sl

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
```

Comment:
This file should be in the same dir as the standard obs_im calc.

But I'm not sure if this is the right folder for both as these are generally automated scripts. But pure naming wise this seems like the right place for them.

Reply:
i would suggest moving the standard obs_im calc script to here as well.
as they can only be submitted on nesi at the moment, until we give it some treatment to adjust headers accordingly.